### ISSUE TITLE
Topic HSBC couldn't be parsed

### ISSUE THREAD

**Issue by yanosh-igor** [spencermountain/wtf_wikipedia#222]
Hi, 
trying to parse topic HSBC
but got errors 
```
TypeError: Cannot read property 'toLowerCase' of undefined
at Object.lcfirst (/app/node_modules/wtf_wikipedia/src/templates/formatting/misc.js:155:20)
    at parseTemplate (/app/node_modules/wtf_wikipedia/src/templates/parse.js:33:30)
    at templates.top.forEach.tmpl (/app/node_modules/wtf_wikipedia/src/templates/index.js:35:12)
    at Array.forEach (<anonymous>)
```

<img width="835" alt="2018-12-15_0030" src="https://user-images.githubusercontent.com/435827/50033365-e4321c00-0000-11e9-9240-554c807a1b5d.png">

**Comment by yanosh-igor** [comment]
Similar issue with topic List of Harvard Crimson men's ice hockey seasons

**Comment by spencermountain** [comment]
fixed it in `7.2.1`
